src/main/java/com/onarandombox/MultiverseCore/commandtools/flags/MVFlag.java

```diff
@@ -0,0 +1,93 @@
+package com.onarandombox.MultiverseCore.commandtools.flags;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents a flag.
+ */
+public class MVFlag {
```

Comment:
Kind of leaning towards doing away with `MV` prefixes on class names.

Reply:
Maybe `CmdFlag` or `CommandFlag`?

Reply:
Definitely `CommandFlag` rather than `CmdFlag`. Abbreviations in names should be avoided in general.